### DISCUSSION TITLE
Whether the u and z_u here should be sorted by time

### DISCUSSION THREAD

**Discussion by ShaneChen1109** [idriskb/NeuroVelo#8] [Q&A]
Dear NeuroVelo creators, Hi! 
I felt confuced about a question when reading the code. Whether the u and z_u here should be sorted by time, similarily to "s = s[index],z_s = z_s[index]"?
Thank you very much in advance.
![屏幕截图 2024-02-23 194427](https://github.com/idriskb/NeuroVelo/assets/141048270/baacdfbc-97f8-4a74-95f1-8d854102ce27)

**Comment by idriskb** [comment]
Thank you for pointing this out. It will be corrected and run the same simulations again to check the results.

**Reply by idriskb** [reply]
Now it is fixed. Thank you for pointing this out. Feel free to ask any questions

**Reply by ShaneChen1109** [reply]
Thank you for the reply and the creative work!
